.gon.hcl

```
@@ -0,0 +1,15 @@
+source = ["./dist/macos-x86-64_darwin_amd64_v1/L2-node"]
+bundle_id = "saturn.filecoin.l2-node"
```

What is the naming convention for bundle IDs?

I would expect either `filecoin.saturn.l2-node` or `l2-node.saturn.filecoin`, because L2 Node belongs to Saturn which belongs to Filecoin.

Should we perhaps use the full domain name including `.io` TLD?

```suggestion
bundle_id = "io.filecoin.saturn.l2-node"
```

FWIW, Filecoin Station uses the following id:

```yaml
appId: io.filecoin.station
```

https://github.com/filecoin-project/filecoin-station/blob/1a635aec4dd580f9fa5ef19f968cdc53afe09521/electron-builder.yml#L1

---

In my understanding bundle ID semantics aren't relevant, but I agree it would be better for them to make sense. I also think that mirroring the domain name is nice, so I'm committing your suggestion 🙏